config/locales/de.yml

```diff
@@ -229,25 +269,6 @@ de:
       Herzlich Willkommen bei %{project_name}. Danke, dass Sie an unserer Dialog-Recherche teilnehmen. In den kommenden Wochen nutzen wir Telegram für den Austausch. Ein Bot arbeitet wie ein Postbote für uns und stellt Fragen und Antworten zu. Ihre Nachrichten lesen und beantworten wir aber selbstverständlich persönlich. Wir freuen uns sehr auf den Dialog mit Ihnen!
```

Comment:
This would need to be configurable as well as the welcome message and especially the names of the team members will likely change from project to project.